packages/core/src/recon.ts

```diff
@@ -47,10 +49,39 @@ export class ReconApi implements IReconApi {
   }
 
   async init(): Promise<void> {
+    if (!this.enabled) {
+      return
+    }
+
     this.#url = await this.#config.url
+    await this.registerInterest(Model.MODEL)
+  }
+
+  async registerInterest(model: StreamID): Promise<void> {
+    if (!this.enabled) {
+      this.#logger.imp(`Recon: disabled, not registering interest in model ${model.toString()}`)
+      return
+    }
+
+    try {
+      await this.#sendRequest(this.#url + `/ceramic/subscribe/model/${model.toString()}`, {
+        method: 'GET',
```

Comment:
this feels like it should be a POST.  @dav1do do you know if there's a plan to change the subscribe endpoint to use POST instead of GET?

Reply:
This route is actually deprecated in favor of interests. It's a `POST` to `/interests/{sort_key}/{sort_value}` with optional `controller` and `streamId` query parameters. 

It returns a 204 and then `/feed/events` is used to get the keys.